test/verify/check-setroubleshoot

```diff
@@ -160,13 +164,51 @@ class TestSelinux(MachineCase):
         b.wait_visible("tr.modification-row > td:contains(Allow zebra to write config)")
         b.wait_visible("tr.modification-row > td:contains(fcontext -a -f a -t samba_share_t -r 's0' '/var/tmp/')")
         b.click(".modifications-export")
-        b.wait_in_text("#basic-tabs-pf-pane-shell > pre:nth-child(1)", "boolean -D")
-        b.wait_in_text("#basic-tabs-pf-pane-shell > pre:nth-child(1)", "fcontext -D")
-        b.wait_in_text("#basic-tabs-pf-pane-shell > pre:nth-child(1)", "boolean -m -1 zebra_write_config")
-        b.wait_in_text("#basic-tabs-pf-pane-shell > pre:nth-child(1)", "fcontext -a -f a -t samba_share_t -r 's0' '/var/tmp/'")
-        b.click(".automation-script-modal .fa-clipboard")
+        b.wait_in_text(".automation-script-modal .modal-body section:nth-child(2) pre", "boolean -D")
+        b.wait_in_text(".automation-script-modal .modal-body section:nth-child(2) pre", "fcontext -D")
+        b.wait_in_text(".automation-script-modal .modal-body section:nth-child(2) pre", "boolean -m -1 zebra_write_config")
+        b.wait_in_text(".automation-script-modal .modal-body section:nth-child(2) pre", "fcontext -a -f a -t samba_share_t -r 's0' '/var/tmp/'")
```

Comment:
You repeat these rather complex selectors several times, please put them into a variable. Same for the nth-child(3) below.